production/loki-mixin-compiled-ssd/dashboards/loki-deletion.json

```
@@ -379,7 +379,7 @@
                   "span": 4,
                   "targets": [
                      {
-                        "expr": "node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"compactor\"}",
+                        "expr": "node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"loki\", pod=~\"(loki|enterprise-logs)-read.*\"}",
```

The new Helm chart will name a single binary loki-namespace-0, -1 and so on. So this can be extended to be
```suggestion
                        "expr": "node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{cluster=~\"$cluster\", namespace=~\"$namespace\", container=\"loki\", pod=~\"(loki.*|enterprise-logs)-read.*\"}",
```

---

I'll fix the matcher and regenerate on monday but it will most likely need to bé fixed on all mixins :)

---

I fixed it here https://github.com/grafana/loki/pull/12790/files#diff-fc2a1d92987b116655610f33f52c66483c47809d20c231089742218d5f626c05R32 but this made this PR a whole lot bigger :D